packages/jest-mock/src/index.js

```diff
@@ -319,67 +321,85 @@ class ModuleMockerClass {
         mockState.calls.push(Array.prototype.slice.call(arguments));
         mockState.timestamps.push(Date.now());
 
-        // The bulk of the implementation is wrapped in an immediately executed
-        // arrow function so the return value of the mock function can
-        // be easily captured and recorded, despite the many separate return
-        // points within the logic.
-        const finalReturnValue = (() => {
-          if (this instanceof f) {
-            // This is probably being called as a constructor
-            prototypeSlots.forEach(slot => {
-              // Copy prototype methods to the instance to make
-              // it easier to interact with mock instance call and
-              // return values
-              if (prototype[slot].type === 'function') {
-                const protoImpl = this[slot];
-                this[slot] = mocker.generateFromMetadata(prototype[slot]);
-                this[slot]._protoImpl = protoImpl;
-              }
-            });
-
-            // Run the mock constructor implementation
-            const mockImpl = mockConfig.specificMockImpls.length
-              ? mockConfig.specificMockImpls.shift()
-              : mockConfig.mockImpl;
-            return mockImpl && mockImpl.apply(this, arguments);
-          }
-
-          const returnValue = mockConfig.defaultReturnValue;
-          // If return value is last set, either specific or default, i.e.
-          // mockReturnValueOnce()/mockReturnValue() is called and no
-          // mockImplementationOnce()/mockImplementation() is called after that.
-          // use the set return value.
-          if (mockConfig.specificReturnValues.length) {
-            return mockConfig.specificReturnValues.shift();
-          }
+        // Will be set to the return value of the mock if an error is not thrown
+        let finalReturnValue;
+        // Will be set to the error that is thrown by the mock (if it throws)
+        let thrownError;
+
+        try {
+          // The bulk of the implementation is wrapped in an immediately
```

Comment:
Everything in the `try {}` block was simply indented with no other changes. The diff makes it look much worse than it is :)

Reply:
Append a `?w=1` to the diff URL and font bother :)

Reply:
Thanks for the tip. Would be nicer if this option was exposed in the UI.

Reply:
I can recommend refined github: https://github.com/sindresorhus/refined-github

I have this button: 
![image](https://user-images.githubusercontent.com/1404810/37236654-2c054b3e-240a-11e8-8fdf-bf678c680f29.png)